modules/gdnative/pluginscript/pluginscript_script.cpp

```diff
@@ -83,36 +83,22 @@ StringName PluginScript::get_instance_base_type() const {
 	return StringName();
 }
 
+
```

Comment:
We're not following PEP8 ;) `clang-format` is configured to prevent more than one consecutive empty lines: https://travis-ci.org/godotengine/godot/jobs/392853115

Reply:
![shame](https://i.imgflip.com/2cb52n.jpg)

Reply:
You mean [RNA](https://en.wikipedia.org/wiki/RNA)? :P